sustain-example.md

```diff
@@ -0,0 +1,32 @@
+This markdown file is just as important as a README. It ensures those using this open source software that there are funds coming in as well as a solid back-up plan in case of an debilitating illness, accident, or untimely death.
+
+## Funding
```

Comment:
I know that patreon and open collective are just two examples.
How about adding a link to a 'thank you' page for donors and sponsors. 
I think it is important to show that money is available, not just where to donate.

Reply:
Yes! That is a great idea. I want this (and anything we do) to be **as vendor-neutral as possible.**